src/components/Home/Index.css

```diff
@@ -0,0 +1,96 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import url('http://fonts.cdnfonts.com/css/gill-sans-extrabolddisplay');
+
+body {
+  color: #fff;
+  background-color: #4369b2;
+}
+
+.card {
+  color: #fff;
+  background-color: #4b71b9;
+  opacity: 0.7;
+}
+
+.card:hover {
+  opacity: 0.3;
+}
+
+.card:nth-child(4n + 2),
+.card:nth-child(4n + 3) {
+  background-color: #668dd4;
+  z-index: 2;
+}
+
+.btn {
+  background: none;
+  border: none;
+}
+
+.img {
+  opacity: 0.6;
+  margin-top: 5%;
+  width: 100%;
+}
+
+.region {
+  font-family: 'Lato', sans-serif;
+  font-size: 2.5rem;
+  align-items: center;
+  justify-content: center;
+}
+
+.regionName {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Lato', sans-serif;
+}
+
+.title {
+  background-color: #5888e4;
+  font-family: 'Gill Sans ExtraBoldDisplay', sans-serif;
+}
+
+.details_page,
+.country {
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+}
+
+.banner_details {
+  padding: 24px;
+  display: flex;
+  gap: 30px;
+  text-align: end;
+}
+
+.details_page {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  color: #fff;
+}
+
+.country {
+  display: flex;
+  flex-direction: column;
+}
+
+/* .card-details {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: #fff;
+  border: 1px solid rgba(0, 0, 0, 0.125);
+  border-radius: 0.25rem;
+} */

```

Comment:
- [ ] Kindly remove the commented code because it makes your work look messy.

Reply:
Commnted css code removed. issu resolved